day-to-day.md

```diff
@@ -4,51 +4,47 @@
 
 Your work and activities may change day-to-day, but there are general expectations that remain the same.
 
-### What We Expect
+### We Expect
 
 - Your best effort to do what's best for you and the company
-- Be open, honest, and empathetic
-- Clearly communicate with the team
-- Be understanding and constructive in conversations
+- Openness, honesty, and empathy
+- Clear communication with the team
+- Conversations to be both understanding and constructive
 
-### What We DON'T Expect
+### We DON'T Expect
 
-- You know everything on the first day, or ever for that matter
-- You are at your desk at all times
-- You work as many hours as possible
-- Unquestioning agreement with the way things are
+- That you know everything on the first day, or ever for that matter
+- You to be working at your desk at all times
+- You to be working as many hours as possible
+- Your unquestioning agreement with the way things are
 
 ## Time tracking
 
 We use [Harvest](https://b2io.harvestapp.com/) to track time, and there are [nice apps out there for it](http://lmgtfy.com/?q=harvest+apps) to help you track as you work.
 
-Our benchmark number is between 32 and 40 tracked hours, and we operate in good
-faith that we are doing the best we can for the company without burning out.
-
-> Question: at what point do we, as a company, start pestering people about
-> their hours? Should one make up the time spent in a long lunch that wasn't a
-> meeting?
+In order for us to fullfil our obligations to clients and to best work with your team, we ask that you are present at predictable times. Our benchmark number is between 35 and 40 tracked hours, and we operate in good faith that we are doing the best we can for the company without burning out. However, if you are not meeting expectations for reliable time tracking or attendance, another teammate will reach out to you to ask if the issues impeding reliability are fixable. We don't expect you to be logged in from 9-5, but appropriate and reliable communication is paramount when working with a team and for clients.
 
 - Not sure what to track to? Ask! (Seriously.)
 - We round up or down to the nearest quarter hour to streamline our billing process
 - Everyone can see everyone's time.
-  - Don't panic, no one's stalking you. Except maybe Tony.
+  - Don't panic, no one's stalking you (see code of conduct). Except maybe Tony.
```

Comment:
Link to code of conduct

Reply:
Should we remove the Tony joke? I can read it both as harmless and as an unnecessary tease.

Also, technically the comma should be a semi-colon or a period.

```
Don't panic. No one's stalking you.
```

Reply:
Yeah, I debated this too.  I'm removing it.